src/mlpack/core/cv/metrics/r2_score.hpp

```diff
@@ -53,12 +53,15 @@ class R2Score
    * @param data Column-major data containing test items.
    * @param responses Ground truth (correct) target values for the test items,
    *     should be either a row vector or a column-major matrix.
+   * @param adjR2 Boolean value which specifies whether to calculate adjusted
+   *    R2 or not
```

Comment:
So, I think having this argument as a parameter to `Evaluate()` actually will mean that the cross-validation system can't use it.  Basically, if I specify `R2Score` as the metric to use with `SimpleCV` or `KFoldCV`, I only get to specify the type---there's no way that I can specify whether the `adjR2` value should be set or not when calling `Evaluate()`.  One way to work around that is to make `adjR2` a template parameter, like `template<bool Adjusted = false> class R2Score`.

Let me know if that doesn't make sense---I can try to clarify further.

Reply:
Agreed, That make sense.